lib/api/core/plugins/pluginsManager.js

```diff
@@ -200,6 +201,14 @@ function PluginsManager (kuzzle) {
       kuzzle.funnel.controllers[name] = controller;
     });
   };
+
+  /**
+   * Returns the list of registered passport strategies
+   * @returns {Promise}
+   */
+  this.listStrategies = function pluginListStrategies () {
+    return Promise.resolve({'authenticationStrategies': _.uniq(this.registeredStrategies)});
```

Comment:
Note: `uniq` could be unnecessary here if we forbid to register 2 strategies with the same name (and indeed, we should).